client/src/components/existing-request/existing-request-display.vue

```diff
@@ -478,22 +478,22 @@ export default class ExistingRequestDisplay extends Mixins(NrAffiliationMixin, C
           await newReqModule.patchNameRequestsByAction(action)
           break
       }
-    } else {
-      newReqModule.setActiveComponent('InvalidActionMessage')
     }
+    // else do nothing -- errors are handled by newReqModule
   }
 
   private async refresh (event) {
     this.refreshCount += 1
     this.checking = true
     try {
-      let resp = await newReqModule.getNameRequest(this.nr.id)
+      const resp = await newReqModule.getNameRequest(this.nr.id)
       this.checking = false
-      if (resp.furnished === 'Y') {
+      if (resp?.furnished === 'Y') {
```

Comment:
This is important because, now, getNameRequest() will return a null if it can't fetch the NR.